selfdrive/car/subaru/values.py

```diff
@@ -269,6 +269,7 @@ class CAR(Platforms):
   # We don't get the EPS from non-OBD queries on GEN2 cars. Note that we still attempt to match when it exists
   non_essential_ecus={
     Ecu.eps: list(CAR.with_flags(SubaruFlags.GLOBAL_GEN2)),
+    Ecu.engine: list(CAR),
```

Comment:
`non_essential_ecus` is to allow missing ECUs, these FW versions still return and as such will still be matched on. We should remove the entries from fingerprints.py to achieve what you want. You can add an entry to `extra_ecus` to keep querying the ECUs for debugging purposes.

Reply:
Just to clarify, we should remove all the Subaru engine and transmission fingerprints and Ecu.engine does not need to be in non-essentials?

Reply:
If you look at the current experimental long implementation then we may need to have per-engine acceleration profiles for tuning

Reply:
We can still query them and look at them for tuning. Do you think it's safe to fall back to one tuning if we haven't seen an engine type? Do we know the FW version encoding??

Reply:
Partial yes - I have the engine sizes for all the fw versions I have added to openpilot in my notes and some are also in subaru-community fork git history comments before we removed them to match upstream. Current openpilot long acceleration is a bit slow when compared to stock eyesight but it is usable as default/fallback. Per model/engine acceleration curves would make the acceleration the same as stock eyesight. I have recorded 0-30, 0-50, 0-70 and 0-90 kph for my 2018 XV eyesight (eco, comfort, normal, dynamic). When I also have some other models data with different engines to compare then I can say if we can do one generic acceleration profile or if the per engine tuning is needed.